src/fsharp/FSharp.Core/map.fsi

```diff
@@ -86,6 +92,14 @@ namespace Microsoft.FSharp.Collections
         [<CompiledName("Add")>]
         val add: key:'Key -> value:'T -> table:Map<'Key,'T> -> Map<'Key,'T>
 
+        /// <summary>Returns a new map with the value stored under key changed according to f.</summary>
+        /// <param name="key">The input key.</param>
+        /// <param name="f">The change function.</param>
+        /// <param name="table">The input map.</param>
+        /// <returns>The resulting map.</returns>
+        [<CompiledName("Change")>]
```

Comment:
This is a new FSharp.Core api, and so should have the ExperimentalAttribute applied, whilst in Preview.

https://github.com/dotnet/fsharp/blob/ed9b9b7df908bbc2b90e46de27dc05678a6d787a/src/fsharp/FSharp.Core/quotations.fsi#L108